code/modules/surgery/organs/lavaland.dm

```diff
@@ -109,6 +115,12 @@
 			L.throw_at_fast(throwtarget, 3, 1)
 		next_command = world.time + cooldown_damage
 
+	//WHO ARE YOU?
+	else if(findtext(command, "who are you?") || findtext(command, "say your name") || findtext(command, "state your name"))
+		for(var/mob/living/L in listeners)
+			L.say("[L.real_name]")
```

Comment:
Add a check if someone's a devil (If they have a mind.devilinfo, or something like that, I forget the exact syntax), and if they are, have them state their demonic name instead.  It would be an interesting way of ousting them.